ui/src/app/views/workspace-edit/component.tsx

```diff
@@ -449,78 +532,189 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
+                <LabeledCheckBox label='Geography (e.g. Rural, urban, suburban, etc.)'
+                   value={this.specificPopulationSelected(SpecificPopulationEnum.GEOGRAPHY)}
+                   onChange={v =>
+                     this.updateSpecificPopulation(SpecificPopulationEnum.GEOGRAPHY, v)}
+                   disabled={!this.state.workspace.researchPurpose.population}/>
+              </div>
+              <div style={{display: 'flex', flexDirection: 'column'}}>
+                <LabeledCheckBox label='Disability status'
+                   value={this.specificPopulationSelected(SpecificPopulationEnum.DISABILITYSTATUS)}
+                   onChange={v =>
+                     this.updateSpecificPopulation(SpecificPopulationEnum.DISABILITYSTATUS, v)}
+                   disabled={!this.state.workspace.researchPurpose.population}/>
+                <LabeledCheckBox label='Access to care'
+                   value={this.specificPopulationSelected(SpecificPopulationEnum.ACCESSTOCARE)}
+                   onChange={v =>
+                     this.updateSpecificPopulation(SpecificPopulationEnum.ACCESSTOCARE, v)}
+                   disabled={!this.state.workspace.researchPurpose.population}/>
+                <LabeledCheckBox label='Education level'
+                   value={this.specificPopulationSelected(SpecificPopulationEnum.EDUCATIONLEVEL)}
+                   onChange={v =>
+                     this.updateSpecificPopulation(SpecificPopulationEnum.EDUCATIONLEVEL, v)}
+                   disabled={!this.state.workspace.researchPurpose.population}/>
+                <LabeledCheckBox label='Income level'
+                   value={this.specificPopulationSelected(SpecificPopulationEnum.INCOMELEVEL)}
+                   onChange={v =>
+                     this.updateSpecificPopulation(SpecificPopulationEnum.INCOMELEVEL, v)}
+                   disabled={!this.state.workspace.researchPurpose.population}/>
+                <LabeledCheckBox label='Other'
+                   value={this.specificPopulationSelected(SpecificPopulationEnum.OTHER)}
+                   onChange={v => this.updateSpecificPopulation(SpecificPopulationEnum.OTHER, v)}
+                   disabled={!this.state.workspace.researchPurpose.population}/>
+                <TextInput type='text' autoFocus placeholder='Please specify'/>
+              </div>
             </div>
           </div>
-          <WorkspaceUnderservedPopulation
-            selectedValues={
-              !this.state.workspace.researchPurpose.underservedPopulationDetails ? [] :
-                  this.state.workspace.researchPurpose.underservedPopulationDetails}
-              onChange={v => this.updateUnderservedPopulation(v)}>
-          </WorkspaceUnderservedPopulation>
         </WorkspaceEditSection>
         <WorkspaceEditSection header='Request a review of your research purpose'
                               tooltip={toolTipText.reviewRequest}>
```

Comment:
This appears not to have `text`. Is that intentional?

Reply:
it is, i believe